internal/boxcli/shellenv.go

```diff
@@ -15,7 +15,9 @@ import (
 
 type shellEnvCmdFlags struct {
 	envFlag
-	config            configFlags
+	config configFlags
+	// TODO: is this flag needed in other commands?
+	envForPackageBins bool
```

Comment:
This flag would be appropriate for anything that instantiates print-dev-env (shell, run, shellenv).

I'm not so sure about the name

Reply:
maybe we go for something less descriptive and just document it:

`--profile-only-mode`

Reply:
yeah, that works too. I usually avoid `profile` in names, because its a nix concept that our users are not exposed to or explained in our docs, but if its a hidden flag then it doesn't matter.

Reply:
changed to `--omit-nix-env` since that is more accurate